### PR TITLE
Improve authentication views

### PR DESCRIPTION
**Pull request by PedroFelipe**
- [x] Reorganize CSS files
- [x] Sign Up
- [x] Sign In
- [x] Password Recover
- [x] Resend Confirmation
- [x] Update README with new test username and password
- [x] Fix flash message component

screen shot 2017-01-12 at 11 06 57 am
screen shot 2017-01-12 at 11 07 01 am
screen shot 2017-01-12 at 11 07 04 am
screen shot 2017-01-12 at 11 07 07 am
screen shot 2017-01-12 at 11 07 13 am

### CODE REVIEW

File: app/assets/stylesheets/_authentication.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  &-button-submit {
+    padding: 7px 20px;
+    font-size: 14px;
+    background-color: #2075F3;
+    border-color: #2075F3;
+    border-radius: 0;
+
+    &:hover,
+    &:focus {
+      background-color: darken(#2075F3, 5%);
```
Comment (talyssonoc) [inline app/assets/stylesheets/_authentication.scss]: We should put those colors in variables, what do you think?

File: app/assets/stylesheets/_authentication.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  &-actions {
+    margin: 0;
+    font-size: 14px;
+
+    &-item {
+      padding-bottom: 6px;
+    }
+
+    &-link {
+      color: #5F6769;
```
Comment (talyssonoc) [inline app/assets/stylesheets/_authentication.scss]: Same think here about using a variable.

File: app/assets/stylesheets/_authentication.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+      &:hover,
+      &:focus {
+        color: #555;
```
Comment (talyssonoc) [inline app/assets/stylesheets/_authentication.scss]: Variable too :)

File: app/assets/stylesheets/_icons.scss [hunk 4]
```diff
@@ -1,11 +1,12 @@
-$icons-spacing:2px;
-$icons-collapse-spacing:6px;
-@import"compass/utilities/sprites";
-$icon-layout: smart;
-$icon-sprite-dimensions: true;
+@import "compass/utilities/sprites";
```
Comment (talyssonoc) [inline app/assets/stylesheets/_icons.scss]: We could think about removing Compass eventually.

File: app/assets/stylesheets/_screen.scss [hunk 5]
```diff
@@ -1,30 +1,12 @@
...
-  padding-left:0;
-}
-
-#page-content-wrapper {
+.content-wrapper {
   width: 100%;
   height: 100%;
   position: absolute;
-  padding-top: 20px;
+  background-color: #F7F6F6;
```
Comment (talyssonoc) [inline app/assets/stylesheets/_screen.scss]: Variable 😛

File: app/assets/stylesheets/_top-bar.scss [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+
+      .form-control {
+        width: auto;
+        display: inline;
+      }
+
+      .btn-group {
+        width: auto;
+
+        button {
```
Comment (talyssonoc) [inline app/assets/stylesheets/_top-bar.scss]: We should use a class here.

File: app/assets/stylesheets/_top-bar.scss [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+    font-size: 1.2em;
+
+    div.velocity_override_container {
+      width: 220px;
+      position: absolute;
+      left: 0;
+      top: 30px;
+      background: #efefef;
+      border: 1px solid #ccc;
+      @include border-radius(3px);
```
Comment (talyssonoc) [inline app/assets/stylesheets/_top-bar.scss]: I think it's better to keep all mixins on the top of the rule set, what do you think?

File: app/helpers/flash_helper.rb [hunk 8]
```diff
@@ -6,6 +6,6 @@ module FlashHelper
   }.freeze
 
   def flash_key_to_bootstrap_class(key)
-    CLASSES_MAP[key.to_sym]
+    CLASSES_MAP[key.to_sym] ? CLASSES_MAP[key.to_sym] : CLASSES_MAP[:error]
```
Comment (talyssonoc) [inline app/helpers/flash_helper.rb]: We can use `fetch` here: `CLASSES_MAP.fetch(key.to_sym, CLASSES_MAP[:error])`

File: app/views/application/_flash.html.erb [hunk 9]
```diff
@@ -1,11 +1,15 @@
-<%= content_tag :div, id: 'flash-messages' do %>
-  <% flash.each do |key, value| %>
-    <%= content_tag :div, class: "alert alert-dismissible #{flash_key_to_bootstrap_class(key)}" do %>
-      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-        <span aria-hidden="true">&times;</span>
-      </button>
+<%= content_tag :div, class: 'container' do %>
```
Comment (talyssonoc) [inline app/views/application/_flash.html.erb]: Can't we just use HTML here?

File: app/views/devise/passwords/new.html.erb [hunk 10]
```diff
@@ -1,23 +1,33 @@
...
+      </div>
     </div>
-  </div>
-  <div class="col-sm-12 col-md-6">
-    <%= render partial: "devise/shared/links" %>
+
+    <div class="clearfix"></div>
+
+    <div class="col-sm-12 col-md-6 col-lg-4 col-md-offset-3 col-lg-offset-4">
+      <%= render :partial => "devise/shared/links" %>
```
Comment (talyssonoc) [inline app/views/devise/passwords/new.html.erb]: You can use just `render 'devise/shared/links'` here

File: app/views/application/_flash.html.erb [hunk 11]
```diff
@@ -1,11 +1,15 @@
...
+  <%= content_tag :div, class: 'row' do %>
+    <%= content_tag :div, class: 'col-sm-12 col-md-6 col-lg-6 col-md-offset-3 col-lg-offset-3 flash-message' do %>
```
Comment (daniloisr) [inline app/views/application/_flash.html.erb]: you can change `contect_tag` to just `<div>` here, as they don't require ruby code :)

File: app/assets/stylesheets/_layout.scss [hunk 12]
```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: 'Noto Sans', sans-serif;
+  color: #3F4C5C;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_layout.scss]: Variable here.

File: app/assets/stylesheets/_top-bar.scss [hunk 13]
```diff
@@ -0,0 +1,152 @@
...
+
+  &-logo {
+    position: relative;
+    top: -4px;
+    margin-right: 10px;
+  }
+
+  &-title {
+    font-size: 1.5em;
+    margin: 0.3em 0;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_top-bar.scss]:
```sass
margin: .3em 0;
```

File: app/assets/stylesheets/_top-bar.scss [hunk 14]
```diff
@@ -0,0 +1,152 @@
...
+    position: relative;
+    display: inline;
+    font-size: 1.2em;
+
+    div.velocity_override_container {
+      width: 220px;
+      position: absolute;
+      left: 0;
+      top: 30px;
+      background: #efefef;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_top-bar.scss]: Variable here.

File: app/assets/stylesheets/_top-bar.scss [hunk 15]
```diff
@@ -0,0 +1,152 @@
...
+    }
+
+    #velocity_value {
+      cursor: pointer;
+      font-weight: bold;
+    }
+  }
+
+  div.velocity.fake #velocity_value {
+    color: #f00;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_top-bar.scss]: Variable here.

File: app/assets/stylesheets/form.scss [hunk 16]
```diff
@@ -0,0 +1,26 @@
+.form {
+  &-error {
+    width: 450px;
+    border: 2px solid red;
+    padding: 7px;
+    padding-bottom: 0;
+    margin-bottom: 20px;
+    background-color: #f0f0f0;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/form.scss]: Variable here.

File: app/assets/stylesheets/form.scss [hunk 17]
```diff
@@ -0,0 +1,26 @@
...
+
+    &-title {
+      text-align: left;
+      font-weight: bold;
+      padding: 5px 5px 5px 15px;
+      font-size: 12px;
+      margin: -7px;
+      margin-bottom: 0px;
+      background-color: #c00;
+      color: #fff;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/form.scss]: And here.

File: app/assets/stylesheets/_icons.scss (hunk 4 above)
Comment (PedroFelipe) [inline app/assets/stylesheets/_icons.scss]: agree

File: config/locales/pt-BR.yml [hunk 18]
```diff
@@ -1,9 +1,11 @@
 pt-BR:
   cm-central:  "CM42 Central"
 
-  log out:  "Sair"
-  log in:   "Entrar"
-  sign up:  "Inscreva-se"
+  welcome: "Bem-vindo al CM42 Central"
```
Comment (marcosinger) [inline config/locales/pt-BR.yml]: typo

Comment (PedroFelipe) [inline config/locales/pt-BR.yml]: oops

**Pull request merged**